Example/Tests/ApiTests.m

```diff
@@ -56,7 +56,7 @@
 
   it(@"should create a thread", ^{
     AddThreadConfig_Schema *schema = [[AddThreadConfig_Schema alloc] init];
-    schema.preset = AddThreadConfig_Schema_Preset_Media;
+    schema.preset = AddThreadConfig_Schema_Preset_Blob;
```

Comment:
I changed this to blob because the test run much faster (no encoding needs to happen)